config/settings_data.json

```diff
@@ -1,33 +1,136 @@
 {
-  "current": "Mercury",
+  "current": {
```

Comment:
Why is it done this way?

Reply:
I guess both ways should work. But for wath this changes? @basilkot

Reply:
revert this change